locals.tf

```diff
@@ -81,6 +81,25 @@ locals {
     }
   ] : []
 
+  application_teams_config_map = var.enable_application_teams == true && length(var.application_teams) > 0 ? [
```

Comment:
It would, but I'm not sure it's possible to achieve it with locals this way, as we need to call it from the root module (auth config). Do you have an example in mind that I can learn from?

Reply:
should be resolved now - moved from root locals to module locals, and calling from root locals the module outputs.